doc/nrf/known_issues.rst

```diff
@@ -1853,6 +1853,18 @@ Closing sockets
   Moreover, closing a socket does not properly clean up all memory resources.
   If a socket is opened and closed multiple times, this  might starve the system.
 
+Modem Library
+============
+
+.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-14312: The :c:func:`nrf_recv` crashes occasionally
+  During execution, in rare cases, the :c:func:`nrf_recv` function crashes.
+
+.. rst-class:: v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-13360: When calling :c:func:`nrf_recv`, the socket context crashes if closed by another thread while receiving data
```

Comment:
nit:
```suggestion
NCSDK-13360: When calling :c:func:`nrf_recv`, the socket context crashes if closed by another thread while receiving data.
```

Reply:
First line shouldn't have a full stop, but as far as I can see, the layout is one brief line to describe the problem and then a full description of the problem (with full stop). I shall update it again to align it to this layout.